packages/network/src/logic/receipts/ReceiptStore.ts

```diff
@@ -0,0 +1,34 @@
+import { NodeId } from '../../identifiers'
+import { Receipt } from 'streamr-client-protocol'
+
+function store(key: string, receipt: Receipt, map: Map<string, Receipt[]>): void {
+    if (!map.has(key)) {
+        map.set(key, [])
+    }
+    map.get(key)!.push(receipt)
+}
+
+export class ReceiptStore {
```

Comment:
Do we want to persist these on the disk? If so, who's responsible, network, client, or broker?